src/thunderbots/software/ai/primitive/directwheels_primitive.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+class DirectWheelsPrimitive : public Primitive
+{
+   public:
+    static const std::string PRIMITIVE_NAME;
+    /**
+     * Creates a new Move Primitive
```

Comment:
To my knowledge, "power" here is just the fraction of total power we can apply to the motors/wheels, with 0 being no power (stationary motors) and +/- 255 being the max and min power we can apply (the fastest the wheels can be driving in either direction).

It would be good to have a comment here explaining what "power" is, since it's not very intuitive straight away.

@garethellis0 might also have some comments on how we can improve the "unit name" in this case? I don't have any great suggestions at the moment that aren't disgustingly verbose.

Reply:
Hopefully it's something we could address in the future with an improved radio protocol/bandwidth. For now I agree comments are the best solution.